my-menu-tsps-web/src/main/webapp/components/my-menu-devices-tsps/my-menu-devices-tsps.html

```diff
@@ -0,0 +1,39 @@
+<!-- col-sm-12 is kinda mandatory here to avoid unnecessary scrollbars... -->
+<div class="my-menu-tsps col-sm-12">

```

Comment:
Please add a very simple `<bmc-title>` so future devs see how this works

Reply:
Ok let's use the BMC components <title-section> <title-bar> <title-display>,...